providers/base/units/power-management/jobs.pxu

```diff
@@ -134,6 +134,48 @@ _description:
  VERIFICATION:
      Did the system resume when the lid was opened?
 
+plugin: user-interact-verify
+category_id: com.canonical.plainbox::power-management
+id: power-management/lid_close_suspend_open
+estimated_duration: 20.0
+requires: device.product == 'Lid Switch'
+_description:
+ PURPOSE:
```

Comment:
Instead of using one `_description` with `PURPOSE...STEPS...VERIFICATION` inside, please use `_purpose`, `_steps` and `_verification` as described in [Checkbox documentation about jobs.](https://checkbox.readthedocs.io/en/latest/units/job.html) (the `_` at the beginning just means those fields can be translated)